geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/scalding/package.scala

```diff
@@ -34,16 +39,84 @@ package object scalding {
   type AccTap = Tap[JobConf, KVRecordReader, MutOutputCollector]
   type AccScheme = Scheme[JobConf, KVRecordReader, MutOutputCollector, Array[Any], Array[Any]]
 
+  type GMRecordReader = RecordReader[Text, SimpleFeature]
+  type GMRecordWriter = RecordWriter[Text, SimpleFeature]
+  type GMOutputCollector = OutputCollector[Text, SimpleFeature]
+  type GMTap = Tap[JobConf, GMRecordReader, GMOutputCollector]
+  type GMScheme = Scheme[JobConf, GMRecordReader, GMOutputCollector, Array[Any], Array[Any]]
+
   object ConnectionParams {
-    val ACCUMULO_INSTANCE   = "geomesa.accumulo.instance"
-    val ZOOKEEPERS          = "geomesa.accumulo.zookeepers"
-    val ACCUMULO_USER       = "geomesa.accumulo.user"
-    val ACCUMULO_PASSWORD   = "geomesa.accumulo.password"
-    val AUTHORIZATIONS      = "geomesa.accumulo.authorizations"
-    val VISIBILITIES        = "geomesa.accumulo.visibilities"
-    val FEATURE_NAME        = "geomesa.feature.name"
-    val CATALOG_TABLE       = "geomesa.feature.tables.catalog"
-    val SCAN_RANGES         = "geomesa.scan.ranges"
-    val MOCK                = "geomesa.mock"
+
+    // old arg strings - replaced with standardized input/output ones below
+    @Deprecated val ACCUMULO_INSTANCE_OLD = "geomesa.accumulo.instance"
```

Comment:
I don't think any of our users picked up on our nascent Scalding support.

I'm tempted to say that deprecating this is overkill.